core/src/main/resources/error/error-classes.json

```diff
@@ -155,6 +160,21 @@
           "To convert values from <srcType> to <targetType>, you can use the functions <functionNames> instead."
         ]
       },
+      "CREATE_MAP_KEY_DIFF_TYPES" : {
```

Comment:
Cloud we make `DATA_DIFF_TYPES ` more general and reuse it?

Reply:
<img width="637" alt="image" src="https://user-images.githubusercontent.com/15246973/199219681-f7089110-7701-42e5-82df-d772f9ed4304.png">

Reply:
but
<img width="814" alt="image" src="https://user-images.githubusercontent.com/15246973/199228400-42c1759f-fc16-4205-a052-fc54b1f3d70b.png">
Very hard to reuse it.
Maybe `functionName` pass to: `map's key`, seems unreasonable